src/components/FeatureExportButton/FeatureExportButton.js

```diff
@@ -0,0 +1,223 @@
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import { FaCloudDownloadAlt } from 'react-icons/fa';
+import KML from 'ol/format/KML';
+import Feature from 'ol/Feature';
+import Style from 'ol/style/Style';
+import Circle from 'ol/style/Circle';
+import Icon from 'ol/style/Icon';
+import VectorLayer from '../../VectorLayer';
+import Button from '../Button';
+
+import { kmlStyle } from '../../utils/Styles';
+
+const propTypes = {
+  /**
+   *  Children content of the Feature export button.
+   */
+  children: PropTypes.oneOfType([PropTypes.string, PropTypes.element]),
+
+  /**
+   * CSS class of the Feature export button.
+   */
+  className: PropTypes.string,
+
+  /**
+   * Format to export features (function).
+   * Supported formats: https://openlayers.org/en/latest/apidoc/module-ol_format_Feature-FeatureFormat.html
+   */
+  format: PropTypes.func,
+
+  /** An existing react-spatial VectorLayer, using a valid ol.source.Vector */
+  layer: PropTypes.instanceOf(VectorLayer).isRequired,
+
+  /**
+   * Map projection.
+   */
+  projection: PropTypes.string,
+
+  /**
+   * HTML tabIndex attribute
+   */
+  tabIndex: PropTypes.number,
+
+  /**
+   * Title of the Feature export button.
+   */
+  title: PropTypes.string,
+};
+
+const defaultProps = {
+  children: <FaCloudDownloadAlt focusable={false} />,
+  className: 'tm-feature-export',
```

Comment:
to avoid duplication of css maybe set the default className to `tm-button tm-feature-export `